RecoMET/METFilters/plugins/HcalLaserEventFilter.cc

```diff
@@ -112,8 +112,8 @@ HcalLaserEventFilter::HcalLaserEventFilter(const edm::ParameterSet& iConfig)
   : vetoByRunEventNumber_ (iConfig.getUntrackedParameter<bool>("vetoByRunEventNumber",true))
   , vetoByHBHEOccupancy_  (iConfig.getUntrackedParameter<bool>("vetoByHBHEOccupancy",false))
   , minOccupiedHBHE_            (iConfig.getUntrackedParameter<unsigned int>("minOccupiedHBHE",5000))
-  , vetoByLaserMonitor_         (iConfig.getUntrackedParameter<bool>("vetoByLaserMonitor",false))
-  , minLaserMonitorCharge_      (iConfig.getUntrackedParameter<double>("minLaserMonitorCharge_",1000.))
+  , vetoByLaserMonitor_         (iConfig.getUntrackedParameter<bool>("vetoByLaserMonitor",true))
+  , minLaserMonitorCharge_      (iConfig.getUntrackedParameter<double>("minLaserMonitorCharge",5000.))

```

Comment:
Please use fillDescriptions() [1], so that you'll get rid of these typing mistakes in the future
[1] https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuideConfigurationValidationAndHelp